lib/rubocop/cop/rspec/mixin/top_level_group.rb

```diff
@@ -29,6 +34,7 @@ def on_top_level_example_group(_node); end
         def on_top_level_group(_node); end
 
         def top_level_group?(node)
```

Comment:
```suggestion
         # @private
         # @deprecated <some description on how to replace>
        def top_level_group?(node)
```

This is more a reminder for us not to forget to remove in 4.0
Should we file a ticket?

Reply:
Like an Issue? Or is there an internal ticketing system?

Reply:
I added the `@deprecated` declaration, but I'm still not sure what filing a ticket entails.

Reply:
Yeah, we just use issues. I created a `4.0` milestone and opened #1986.